processing/src/main/java/io/druid/segment/data/GenericIndexedWriter.java

```diff
@@ -129,11 +129,11 @@ private static void writeLongValueToOutputStream(ByteBuffer helperBuf, long valu
     outLong.write(helperBuf.array());
   }
 
-  private static void writeIntValueToOutputStream(ByteBuffer helperBuf, int value, CountingOutputStream outLong)
```

Comment:
If you are making this a publicly reused thing, how about putting it in `SerializerUtils`.  Package private is a false security, you are using it across class boundaries so it might as well be fully public.